src/main/java/net/greatstart/dto/DtoProject.java

```diff
@@ -11,43 +12,11 @@
     private Long id;
     @Valid
     @NotNull
-    private DtoCategory category;
-    @Valid
     private DtoProjectDescription desc;
+    @Valid
+    private Category category;
     private DtoUserProfile owner;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "owner" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)